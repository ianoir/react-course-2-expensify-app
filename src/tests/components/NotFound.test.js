import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFound';

// Should render the not found page
test('should render the not found page', () => {
    const wrapper = shallow(<NotFoundPage /> );
    expect(wrapper).toMatchSnapshot();
});