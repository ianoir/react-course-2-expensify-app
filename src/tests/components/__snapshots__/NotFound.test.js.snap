// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the not found page 1`] = `
<div>
  Error 404! - 
  <Link
    replace={false}
    to="/"
  >
    Go to Home
  </Link>
</div>
`;
